src/utils/config/template/question.template.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class QuestionTemplate {
+  @Field(() => String, {
+    nullable: false,
+    description: 'Question.',
```

Comment:
That's a bit laconic - no description will be better than this description (in my opinion)